core-api/src/main/java/org/silverpeas/core/util/DateUtil.java

```diff
@@ -797,8 +783,8 @@ public static Date parseISO8601Date(final String date) throws ParseException {
    * @param withTime the time in the Java date has to be taken into account.
    * @return a Temporal object.
    */
-  public static Temporal<?> asTemporal(final java.util.Date aDate, boolean withTime) {
-    Temporal<?> temporal;
+  public static Temporal asTemporal(final java.util.Date aDate, boolean withTime) {
+    Temporal temporal;
     if (withTime) {
       temporal = new DateTime(aDate);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "DateTime"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)